apps/backoffice-v2/src/pages/Entity/components/Case/Case.Content.tsx

```diff
@@ -24,7 +24,7 @@ export const Content: FunctionComponent<PropsWithChildren> = ({ children }) => {
           opacity: 1,
         }}
         transition={{ duration: 0.3 }}
-        className={`mx-2 flex flex-col space-y-3 p-1 pt-2`}
+        className={`mx-2 flex h-full flex-col space-y-3 p-1 pt-2`}
```

Comment:
Do the non-merchant monitoring changes remain the same in visuals?

Reply:
P3

Reply:
Havent seen that layout breaks.